frontend/service.go

```diff
@@ -576,6 +576,16 @@ func (s *lwdStreamer) GetMempoolTx(exclude *walletrpc.GetMempoolTxRequest, resp
 			return status.Errorf(codes.InvalidArgument, "exclude txid %d is larger than 32 bytes", i)
 		}
 	}
+	if slices.Contains(exclude.PoolTypes, walletrpc.PoolType_POOL_TYPE_INVALID) {
```

Comment:
@LarryRuane I'm implementing this on Zaino and I realized that GetMempoolTxRequest can also contain _uneven_ number of bytes that is less than 32 bytes and according to how suffixes are required to be, then they can't be an _uneven_ number of bytes to reverse. 

> /// A list of transaction ID byte string suffixes that should be excluded
    /// from the response. These suffixes may be produced either directly from
    /// the underlying txid bytes, or, if the source values are encoded txid
    /// strings, by truncating the hexadecimal representation of each
    /// transaction ID to an even number of characters, and then hex-decoding
    /// and then byte-reversing this value to obtain the byte representation.

Reply:
That's why I added the reserved field, but I don't think it's worth doing now, particularly given that I don't think that `GetMempoolTx` is as widely used as the streaming noncompact version. Just having documented the semantics clearly here is sufficient.